### PR TITLE
add patch for Solution, fix various tests

### PR DESCRIPTION
**Pull request by dougbrad**
…and ManagementConfiguration API will be deprecated in future so in a couple of cases removed inputs

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json [hunk 1]
```diff
@@ -821,6 +877,19 @@
...
+            "properties": {
+                "tags": {
+                    "type": "object",
+                    "additionalProperties": {
+                      "type": "string"
+                    },
+                    "description": "Resource tags"
+                  }
+            },
+            "description": "The properties of a Soluton that can be patched.",
```
Comment (nschonni) [inline specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]:
```suggestion
            "description": "The properties of a Solution that can be patched.",
```

**Comment by tjprescott** [comment]
@dougbrad can you elaborate why you opened this as a draft PR?

**Review by ericc1103 — APPROVED** [review]
Look good on PATCH related.

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json (hunk 1 above)
Comment (dougbrad) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: Thanks, fixed

**Comment by dougbrad** [comment]
The OpenAPI tests cannot be run, forever get:
  Still working!

**Comment by dougbrad** [comment]
> @dougbrad can you elaborate why you opened this as a draft PR?

**Comment by dougbrad** [comment]
> @dougbrad can you elaborate why you opened this as a draft PR?

Hi, it was just so our team could do further review. Should have done that before-hand though.

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/examples/ManagementConfigurationCreate.json [hunk 2]
```diff
@@ -5,51 +5,7 @@
...
-                        {
-                            "name": "jsonobject",
-                            "value": {
-                                "displayName": "abcde",
-                                "query": "hello"
-                            }
-                        }
-                ]
-            }
+            "location":"East US"
```
Comment (tjprescott) [inline specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/examples/ManagementConfigurationCreate.json]: This create requires no parameters?

**Review by tjprescott — COMMENTED** [review]
It seems the response bodies of a lot of examples have been slashed. Is this because the previous examples were incorrect and did not represent the actual service response?

**Review by paruizfi — APPROVED** [review]

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/examples/ManagementConfigurationCreate.json (hunk 2 above)
Comment (dougbrad) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/examples/ManagementConfigurationCreate.json]: These are non-used API's, created before a re-org. We plan to remove them after doing a little more diligence.
So took quickest rout in order to get the test passing.

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json [hunk 3]
```diff
@@ -84,6 +84,55 @@
...
+                        "$ref": "#/parameters/SolutionNameParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/SolutionPatch"
+                        },
+                        "description": "The parameters required to patch a Solution."
```
[the remark this answers is not in the document]
Comment (dougbrad) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: Just tags

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dougbrad) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: Fixed (review was of older version)

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json [hunk 4]
```diff
@@ -84,6 +84,55 @@
                     }
                 }
             },
+            "patch": {
+                "tags": [
+                    "Solution"
+                ],
+                "x-ms-examples": {
+                    "SolutionCreate": { "$ref": "./examples/SolutionUpdate.json" }
```
[the remark this answers is not in the document]
Comment (dougbrad) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: Fixed

**Review by tjprescott — APPROVED** [review]

**Pull request merged**